### ISSUE TITLE
LLVM error after build pytorch and torch_tvm from source

### ISSUE THREAD

**Issue by DreamChaserMXF** [pytorch/tvm#141]
Environments: Ubuntu16.04  LTS, Python 3.6.9, LLVM 8.0, GCC 5.4, Pytorch Nightly version (build from source), torch_tvm master branch.

Successfully import tvm, but encounter error when import torch_tvm:

![LLVM-error](https://user-images.githubusercontent.com/16394892/68832881-050d6080-06ed-11ea-85a7-5ec8a55b8660.png)

below is __init__.py of package torch_tvm.

Anyone has any advice?

**Comment by yinghai** [comment]
Does this clash with `import tvm`?

**Comment by DreamChaserMXF** [comment]
> Does this clash with `import tvm`?

No, it doesn't. `import tvm` can be executed without any error and warning.
It seems like `tvm` was successfully compiled, but torch_tvm was somehow corrupted.

**Comment by yinghai** [comment]
@bwasti Anything on top of your head?

**Comment by IHIaadj** [comment]
Same issue here on Ubuntu 18.04.

**Comment by MaxBareiss** [comment]
I also have this issue on ~~18.04~~16.04 with LLVM 9.0.0. It appears to be related to this issue: https://github.com/ziglang/zig/issues/67, which also appears elsewhere in different forms.
